libs/common-utils/src/environments.ts

```diff
@@ -45,7 +47,9 @@ const { isLocal, isDev, isPr, isStaging, isProd, isEns, isBarn } = checkEnvironm
   window.location.pathname
 )
 
-export const environmentName = (function () {
+export type EnvironmentName = 'local' | 'development' | 'pr' | 'staging' | 'production' | 'barn' | 'ens'
```

Comment:
Can this type be reused instead of `type Envs`?
It has the same info.

Reply:
https://github.com/cowprotocol/cowswap/pull/3312